geode-redis/src/main/java/org/apache/geode/redis/internal/pubsub/PubSubImpl.java

```diff
@@ -140,18 +139,17 @@ public long punsubscribe(GlobPattern pattern, Client client) {
   }
 
   @Override
-  public List<byte[]> findSubscribedChannels(Client client) {
+  public List<byte[]> findSubscriptionsNames(Client client) {
     return subscriptions.findSubscriptions(client).stream()
-        .map(Subscription::getChannelName)
-        .filter(Objects::nonNull)
+        .map(Subscription::getSubscriptionName)
         .collect(Collectors.toList());
   }
 
   @Override
-  public List<byte[]> findSubscribedPatterns(Client client) {
+  public List<byte[]> findSubscriptionsNames(Client client, Subscription.Type type) {
     return subscriptions.findSubscriptions(client).stream()
-        .map(Subscription::getPatternName)
-        .filter(Objects::nonNull)
+        .filter(s -> s.getType().equals(type))
```

Comment:
Also no biggy, but for enum equality it's slightly better to use `==` since it's also null safe.

Reply:
Good to know! I'll update it.